Zend/zend_operators.c

```diff
@@ -30,6 +30,8 @@
 #include "zend_strtod.h"
 #include "zend_exceptions.h"
 #include "zend_closures.h"
+#include "zend_interfaces.h"
+#include "ext/spl/spl_iterators.h"
```

Comment:
hmm, I don't think include a spl/ file in Zend is a good idea...what do you need please port it into Zend